ddtrace/contrib/psycopg/patch.py

```diff
@@ -10,23 +11,32 @@
 # Original connect method
 _connect = psycopg2.connect
 
-def patch():
+
+def patch(tracer=None):
     """ Patch monkey patches psycopg's connection function
         so that the connection's functions are traced.
     """
-    wrapt.wrap_function_wrapper(psycopg2, 'connect', patched_connect)
-    _patch_extensions() # do this early just in case
+    if getattr(psycopg2, '_datadog_patch', False):
+        return
+    setattr(psycopg2, '_datadog_patch', True)
+
+    wrapt.wrap_function_wrapper(psycopg2, 'connect', functools.partial(patched_connect, tracer=tracer))
```

Comment:
I think you don't need the `functools.partial` if you don't pass the `tracer=` right?